src/bp-settings/classes/class-bp-rest-account-settings-options-endpoint.php

```diff
@@ -752,7 +752,7 @@ public function get_notifications_fields() {
 
 			$fields_forums[] = array(
 				'name'        => 'notification_forums_following_topic',
-				'label'       => __( 'A member creates discussion in a forum you are subscribed', 'buddyboss' ),
+				'label'       => __( 'A member creates a discussion in a forum you are subscribed to', 'buddyboss' ),
```

Comment:
esc_html__( 'A member creates a discussion in a forum you are subscribed to', 'buddyboss' ),